yast/package/yast2-smt.spec

```diff
@@ -44,8 +44,8 @@ Requires:	/usr/bin/getent
 Requires:       sudo
 # Modified smt-catalogs (added --batch-mode)
 # SMT::Client::getPatchStatusLabel returning two values
-# require smt version having SMT::Curl
-Requires:	smt >= 1.1.23
+# don't require SMT (instead ask to install it), but prevent older versions
+Conflicts:	smt < 1.1.23
```

Comment:
Is `1.1.23` really ok? Does it have scc support? i.e. `src/modules/SMTData.rb` uses smt-scc-sync script added which is AFAIK `2.0.1`